trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -0,0 +1,35 @@
+const { param } = require('express-validator');
+
+const userParamNumericValidation = param('user').isAlphanumeric().withMessage('The User ID (user) provided should be numeric');
+const facilityIdNumericValidation = param('facilityId').isNumeric().withMessage('The Facility ID (facilityId) provided should be numeric');
```

Comment:
isMongoId()

Reply:
So does the frontend pass around the mongo Object IDs rather than say facility IDs that looks more like `0030605900` (with the first 3 digits depending on environment?)